Sources/BraintreePayPalMessaging/BTPayPalMessagingOfferType.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import PayPalMessages
+
+/// Preferred message offer to display
+/// - Note: This module is in beta. It's public API may change or be removed in future releases.
+public enum BTPayPalMessagingOfferType {
+
+   /// Pay Later short term installment
```

Comment:
We're eventually going to want docs that clearly explain the difference between short term and long term that we reference or copy/paste the definitions into our comments

Reply:
Agree, [JS doesn't have the same options for offer type](https://developer.paypal.com/docs/checkout/pay-later/us/integrate/customize-messages/) so just went with what the Messages team had documented for now

Reply:
😬 ok, we're going to want to get JS in parity with us. It looks like JS is also using the description `none` for the simple logo, so we may want to take to JS and PP Messaging folks about that choice, too.

I'll circle this feedback to Product, we can resolve this convo